README.md

```diff
@@ -23,7 +23,7 @@ Install-Package Auth0.ManagementApi
 
 ### Usage
 
-Generate a token for the API calls you wish to make (see [https://auth0.com/docs/api/v2/tokens](https://auth0.com/docs/api/v2/tokens)). Create an instance of the `ManagementApiClient` class with the token and the API URL of your Auth0 instance:
+Generate a token for the API calls you wish to make (see [How to Get an Access Token for the Management API](https://auth0.com/docs/api/v2/tokens)). Create an instance of the `ManagementApiClient` class with the token and the API URL of your Auth0 instance:
```

Comment:
This points to a 404. Can you update to https://auth0.com/docs/api/management/v2/tokens?